Xamarin.Essentials/Clipboard/Clipboard.mac.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Threading.Tasks;
+using AppKit;
+using Foundation;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Clipboard
+    {
+        static readonly string pasteboardType = NSPasteboard.NSPasteboardTypeString;
```

Comment:
This code should use `NSString.GetReadableTypesForPasteboard` instead of hardcoding just this one format for reading. Pasteboards can contain UTF-16 text instead of UTF-8, etc. It's best to support whatever NSString supports.

When writing to the pasteboard, it's fine to choose your own format so long as it's in the readable formats list. I recommend `public.utf8-plain-text` but `NSPasteboard.NSPasteboardTypeString` is fine.

Reply:
I ended up with this:

```csharp
static Task<string> PlatformGetTextAsync()
{
    var strs = Pasteboard.ReadObjectsForClasses(
        new ObjCRuntime.Class[] { new ObjCRuntime.Class(typeof(NSString)) },
        null);

        return Task.FromResult(strs?[0]?.ToString());
}
```

I did test to ensure it can retrieve a utf16 string which it does.  I think we're good here!